lib/features/shalat/data/models/prayer_schedule_setting_model.codegen.dart

```diff
@@ -59,7 +59,9 @@ abstract class PrayerAlarmModel with _$PrayerAlarmModel {
   const factory PrayerAlarmModel({
     @HiveField(0) DateTime? time,
     @HiveField(1) String? prayer,
-    @HiveField(2) @Default(false) bool isAlarmActive,
+    @HiveField(3) @Default(3) dynamic alarmType,
```

Comment:
kalau memang ini integer ganti dynamic jadi int
```suggestion
    @HiveField(3) @Default(3) int alarmType,
```